src/main/scala/vsys/settings/WebhookEventSettings.scala

```diff
@@ -0,0 +1,98 @@
+package vsys.settings
+
+trait WebhookEventSettings{
+  val typeId: Int 
+  val typeDescription: String
+}
+
+case class BlockAppendedEventSettings(
+  withTxs: Boolean, 
+  afterHeight: Long, 
+  afterTime: Long
+) extends WebhookEventSettings {
+  override val typeId = 1
+  override val typeDescription = "Block Appended"
```

Comment:
may consider add generator to the setting

Reply:
what is a generator for these classes?

Reply:
just similar like `relateAcc` in `TxConfirmedEventSettings` and `generator` is the minter of the block. however, i still consider whether we need this element here.